assets/app/views/_alerts.html

```diff
@@ -10,7 +10,7 @@
     </div>  
     <div ng-switch-when="success" class="alert alert-success">
       <span class="pficon pficon-ok"></span>
-      <strong ng-if="alert.message" style="margin-right: 5px;">{{alert.message}}</strong><span ng-if="alert.details">{{alert.details}}</span>
+      <strong ng-if="alert.message" style="margin-right: 5px;">{{alert.message}}</strong><span ng-if="alert.details">{{alert.details}}</span><a ng-if="alert.link" class="alert-link" href="{{alert.link}}">View details</a>
```

Comment:
"View details" may not always make sense, this text should really be an option you are passing in, and then have some sane default in case they dont pass it in, in this case View details might be ok as the default.